src/stamps.ts

```diff
@@ -182,6 +184,26 @@ export class StampsManager {
     throw new Error('No postage stamp')
   }
 
+  public async refreshStampsReupload(beeApi: Bee): Promise<void> {
```

Comment:
Ohhh now I understand the `if` in `index.ts`. Yeaaah I would not do this. Please keep all the reupload functionality in the `ContentManager` and separate it from the `StampsManager`. It is cleaner and you don't have to duplicate the functionality.